lib/src/widgets/help_collection/first_help_widget.dart

```diff
@@ -0,0 +1,53 @@
+import 'package:flutter/material.dart';
+import 'package:help_ukraine_widget/help_ukraine_widget.dart';
+
+class FirstHelpWidget extends StatelessWidget {
+  ///Constructor
+  const FirstHelpWidget({Key? key}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return HelpWidget(
+      mainCardTitle: 'Stop War! You can help!',
+      options: defaultOptionsList,
+      optionsCardBuilder: (options, onClose) {
+        return CardRounded(
+          key: const ValueKey(2),
+          child: LinksCardWidget(
+            options: options,
+            onHidePressed: onClose,
+          ),
+        );
+      },
+      mainCardBuilder: (flag, title, desc, onDetails, onClose) {
```

Comment:
Oof, that's a lot of arguments for a builder.

Reply:
I wonder if we could simplify this.

Reply:
I think we can:

1. Let's remove the `flag` arg -- it's much better to just have the default flag as a separate Widget anyone can use.
2. Let's remove `title` and `desc` -- providing those from within the widget is unnecesssary.
3. With the above, we're only left with the callbacks -- `onDetails` and `onClose`.
4. For the callbacks -- let's mimic `TextEditingController`. `HelpWidget` will have an argument -- `HelpWidgetController controller`, which will have a few methods like `close` and `showDetails`, which could be used to control a custom-made widget. This will make for a much better separation of concerns, and make the code easier to modify.
5. Extra -- let's make the transition animation builder external as well.